assets/assetpack.def

```diff
@@ -168,62 +168,68 @@
 ! logo-16.png
 < images/logo-16.png
 
+! logo-none-16.png
+< images/logo-16.png
+
+! logo-scheduled-16.png
+< images/logo-scheduled-16.png
+
 ! logo-blocked-16.png
 < images/logo-blocked-16.png
 
+! logo-execution-16.png
+< images/logo-execution-16.png
```

Comment:
`ASSIGNED`, `SETUP`, `RUNNING` and `UPLOADING` are "execution states". I made it explicit and use the meta state `EXECUTION` here (instead of mapping all of these to `RUNNING` like before).

Reply:
I see. Because we use participles for most others, how about calling it "executing"?

Reply:
nevermind, it's fine